authentik/core/api/users.py

```diff
@@ -765,6 +766,8 @@ def partial_update(self, request: Request, *args, **kwargs) -> Response:
         response = super().partial_update(request, *args, **kwargs)
         instance: User = self.get_object()
         if not instance.is_active:
+            RefreshToken.objects.filter(session__user=instance).delete()
```

Comment:
This should be in a signal in the oauth2 provider app, as a user might get deactivated through other means (and we shouldn't introduce new direct app dependencies if possible)

Reply:
any other places where this signal should be sent from?